source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -6018,6 +6018,9 @@ class ActionCollection {
                 action.render();
 
                 if (action.renderedElement) {
+                    if (_isDesignMode) {
+                        action.renderedElement.tabIndex = -1;
+                    }
```

Comment:
this is goodness. _however_ it means that we need a keyboard-accessible way to expand an `Action.ShowCard` in design mode (feel free to file an issue to revisit this later if that makes more sense to you)

Reply:
Just brough in some changes from [this PR](https://github.com/microsoft/AdaptiveCards/pull/7159) which should fix this!